data.sql

```diff
@@ -4,6 +4,13 @@ INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
 INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Gabumon', date '11-15-2018', '8', 'true', '2');
 INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Pikachu', date '01-17-2021', '15.04', 'false', '1');
 INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Devimon', date '05-12-2017', '11', 'true', '5');
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Charmander', date '02-08-2020', '11', 'false', '0');
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Plantmon', date '11-15-2021', '5.7', 'true', '2');
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Squirtle', date '05-02-1993', '12.13', 'false', '3');
```

Comment:
- [ ] @Nejjari001 you have added the wrong date of birth and  weight values kindly correct it.